projects/conding-challenge-i/src/App.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react'
+import './App.css'
+
+import CardName from './components/CardName'
+
```

Comment:
```suggestion
// Importaciones de React hasta el Top
import { useState } from 'react'
// Importaciones de librerias de terceros, etc...

// Importacion de componentes y elementos propios del proyecto
import CardName from './components/CardName'

//Importacion de estilos y extras (.css, imagenes, etc...)
import './App.css'


```

Esto es mas un tema de organización de dependencias, solo es una sugerencia para que puedas mantener organizadas tus dependencias y sea mas facil encontrarlas cuando son demasiadas.